src/webob/acceptparse.py

```diff
@@ -686,10 +686,16 @@ def basic_filtering(self, language_tags):
         suggested as a matching scheme for the ``Accept-Language`` header in
         :rfc:`RFC 7231, section 5.3.5 <7231#section-5.3.5>`, and defined in
         :rfc:`RFC 4647, section 3.3.1 <4647#section-3.3.1>`. It filters the
-        tags in the `language_tags` argument, and returns a list of the tags
-        that match the language ranges in the header according to the Basic
-        Filtering matching scheme, in descending order of preference, together
-        with the qvalue of the range each tag matched.
+        tags in the `language_tags` argument and returns the ones that match
```

Comment:
I think the idea of the single backticks in Sphinx and the `default_role` is that we wouldn't need to specify a role &mdash; there's an example [here](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-any), from one of the links in the note. So if they are cross-references, `<cite>` kind of makes sense? But I don't think that works for parameters like `language_tags`, so it doesn't help us there.

Italics kinda makes sense to me for parameters, although `<cite>` doesn't without cross-references. If you'd prefer them styled differently, maybe we can look into it after GSoC?